src/translate.ts

```diff
@@ -56,7 +56,8 @@ export async function translate(
   lang: string,
   model: string,
   translations: GetTextTranslation[],
-  contextFile: string
+  contextFile: string,
+  timeout: number = parseInt(process.env.API_TIMEOUT || "20000")
```

Comment:
This line of default value is redundant
```suggestion
  timeout: number
```